### PR TITLE
FIx lp8 driver

### PR DESCRIPTION
**Pull request by blavka**

### CODE REVIEW

File: bcl/src/bc_lp8.c [hunk 1]
```diff
@@ -74,16 +77,13 @@ bool bc_lp8_get_concentration_ppm(bc_lp8_t *self, float *ppm)
 {
     if (!self->_valid)
     {
+    	*ppm = NAN;
```
Comment (hubpav) [inline bcl/src/bc_lp8.c]: Bad formatting :-)

File: bcl/src/bc_lp8.c [hunk 2]
```diff
@@ -5,7 +5,10 @@
 #define _BC_LP8_MODBUS_READ 0x44
 #define _BC_LP8_INITIAL_MEASUREMENT 0x10
 #define _BC_LP8_SEQUENTIAL_MEASUREMENT 0x20
-#define _BC_LP8_RX_ERROR_STATUS0 (3 + 39)
+#define _BC_LP8_RX_ERROR_STATUS0 (3 + 0xa7 - 0x80)
+#define _BC_LP8_RX_ERROR_STATUS1 (3 + 0xa6 - 0x80)
+#define _BC_LP8_RX_CONC (3 + 0x9a - 0x80)
```
Comment (hubpav) [inline bcl/src/bc_lp8.c]: What is CONC?

Comment (blavka) [reply bcl/src/bc_lp8.c]: Datasheat LP8, page 17, table Parameters, row 4 : 
Conc : Non pressure-corrected unfiltered concentration value

Comment (hubpav) [reply bcl/src/bc_lp8.c]: OK

**Pull request merged**